src/lib/feature-finder.js

```diff
@@ -59,8 +62,8 @@ function parseFeature(featurePath) {
     let file = fs.readFileSync(featurePath, {encoding: 'utf8'});
     return gherkinParser.parse(file).feature;
   } catch (e) {
-    console.log(featurePath + ' could not be parsed from Gherkin, ignoring as a feature file.', e);
-    return {children: []};
+    console.error(featurePath + ' could not be parsed from Gherkin, ignoring as a feature file.\n', e.message);
```

Comment:
The stack trace in this case isn't very helpful as it's all in the gherkin project. What's important is the message that gherkin generates.